src/Esfa.Vacancy.Api.Types/ApprenticeshipVacancy.cs

```diff
@@ -186,5 +186,20 @@ public class ApprenticeshipVacancy
         /// Phone number of the contact person
         /// </summary>
         public string ContactNumber { get; set; }
+
+        /// <summary>
+        /// The training provider's trading name
+        /// </summary>
+        public string TrainingProvider { get; set; }
+
+        /// <summary>
+        /// The training provider's UKPRN
+        /// </summary>
+        public string TrainingProviderUkprn { get; set; }
+
+        /// <summary>
+        /// The training provider site
```

Comment:
The training provider site's trading name

Reply:
Site's trading name doesn't sound correct to me. Perhaps the name should reflect the value that is expected by the consumer, which we are not sure of, so I think it's ok as is.

Reply:
Provider Site 'trading name' is a hangover from EDRS.  As this is really just a location I'm not sure we should refer to a trading name.  So although this field is populated from the 'TradingName' field in the DB I'm not sure there is any value in exposing this in the documentation.